README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
   }
   ```
 
-## Подробнее
+## Details
 
-### Специфика БЭМ-проектов
-[БЭМ](https://bem.info) (Блок-Элемент-Модификатор) - это придуманная в [Яндексе](https://yandex.ru) методология разработки веб-приложений, в основе которой лежит компонентный подход. БЭМ - это также набор инструментов для удобной разработки в соответствии с принципами методологии. БЭМ помогает быстрее разрабатывать сайты и поддерживать их долгое время.
+### Specificity of BEM projects
+[BEM](https://en.bem.info) (Block-Element-Modifier) is the frontend development methodology, which was created in [Yandex](https://yandex.com/company). It is based on the component approach. BEM is also the set of tools for comfortable development according to principles of methodology. BEM hepls to develop the sites faster and support them for a long time.
```

Comment:
[BEM](https://en.bem.info) (Block-Element-Modifier) is a frontend development methodology developed at [Yandex](https://yandex.com/company). It is based on the component approach. BEM is also a set of tools that make it easy to apply BEM principles during development. You can use BEM for faster development and long-term maintenance of sites.